src/diffusers/models/attention_processor.py

```diff
@@ -68,7 +69,6 @@ def __init__(
         cross_attention_dim = cross_attention_dim if cross_attention_dim is not None else query_dim
         self.upcast_attention = upcast_attention
         self.upcast_softmax = upcast_softmax
-        self.cross_attention_norm = cross_attention_norm
```

Comment:
If a layer's existence is configured with a boolean flag, we usually define the layer as none or defined, and then check if the layer is not none in the forward method. I consolidated this check in the `prepare_encoder_hidden_states` method and removed all references to the boolean

Reply:
ok for me